tests/dataframe_test.py

```diff
@@ -0,0 +1,274 @@
+import pandas
+
+from base_test import ArkoudaTest
+from context import arkouda as ak
+
+class DataFrameTest(ArkoudaTest):
+    def test_dataframe_creation(self):
+        df = ak.DataFrame()
+        self.assertIsInstance(df, ak.DataFrame)
+
+        username = ak.array(['Alice', 'Bob', 'Alice', 'Carol', 'Bob', 'Alice'])
+        userid = ak.array([111, 222, 111, 333, 222, 111])
+        item = ak.array([0, 0, 1, 1, 2, 0])
+        day = ak.array([5, 5, 6, 5, 6, 6])
+        amount = ak.array([0.5, 0.6, 1.1, 1.2, 4.3, 0.6])
+        df = ak.DataFrame({'userName': username, 'userID': userid,
+                           'item': item, 'day': day, 'amount': amount})
+        self.assertIsInstance(df, ak.DataFrame)
+        self.assertEqual(df[0], {'index': 0, 'userName': 'Alice', 'userID': 111, 'item': 0, 'day': 5, 'amount': 0.5})
+        self.assertTrue((df['userName'] == ak.array(['Alice', 'Bob', 'Alice', 'Carol', 'Bob', 'Alice'])).all())
+        self.assertEqual(len(df), 6)
+
+    def test_drop(self):
+        username = ak.array(['Alice', 'Bob', 'Alice', 'Carol', 'Bob', 'Alice'])
+        userid = ak.array([111, 222, 111, 333, 222, 111])
+        item = ak.array([0, 0, 1, 1, 2, 0])
+        day = ak.array([5, 5, 6, 5, 6, 6])
+        amount = ak.array([0.5, 0.6, 1.1, 1.2, 4.3, 0.6])
+        df = ak.DataFrame({'userName': username, 'userID': userid,
+                           'item': item, 'day': day, 'amount': amount})
+        df.drop('userName')
+        self.assertEquals(df.__str__(), ak.DataFrame({'userID': userid,
+                                           'item': item, 'day': day, 'amount': amount}).__str__())
```

Comment:
we might be able to compare directly against pandas instead of what we think the output should be. Also we might be able to pull out some of the repeated set up

something like the below
```python
from base_test import ArkoudaTest
   def set_up(self):
        ArkoudaTest.setUp(self)
        username = ['Alice', 'Bob', 'Alice', 'Carol', 'Bob', 'Alice']
        userid = [111, 222, 111, 333, 222, 111]
        item = [0, 0, 1, 1, 2, 0]
        day = [5, 5, 6, 5, 6, 6]
        amount = [0.5, 0.6, 1.1, 1.2, 4.3, 0.6]
        ak_df = ak.DataFrame({'userName': ak.array(username), 'userID': ak.array(userid),
                           'item': ak.array(item), 'day': ak.array(day), 'amount': ak.array(amount)})
       pd_df = pd.DataFrame({'userName': username, 'userID': userid,
                           'item': item, 'day': day, 'amount': amount})

   def test_drop(self):
        ak_df.drop('userName')
        pd_df.drop('userName')
        # compare the output somehow, probably using `.to_pandas`
```

Reply:
@pierce314159 I agree with you. I will make sure to clean up the tests along with the other sections of DataFrame when we get there. I will add an issue to handle this.